x/committee/types/permissions.go

```diff
@@ -585,3 +616,84 @@ func addressesEqual(addrs1, addrs2 []sdk.AccAddress) bool {
 	}
 	return areEqual
 }
+
+// AllowedMoneyMarket permission struct for money market parameters (hard module)
+type AllowedMoneyMarket struct {
+	Denom                  string `json:"denom" yaml:"denom"`
+	BorrowLimit            bool   `json:"borrow_limit" yaml:"borrow_limit"`
+	SpotMarketID           bool   `json:"spot_market_id" yaml:"spot_market_id"`
+	ConversionFactor       bool   `json:"conversion_factor" yaml:"conversion_factor"`
+	InterestRateModel      bool   `json:"interest_rate_model" yaml:"interest_rate_model"`
+	ReserveFactor          bool   `json:"reserve_factor" yaml:"reserve_factor"`
+	KeeperRewardPercentage bool   `json:"keeper_reward_percentage" yaml:"keeper_reward_percentage"`
+}
+
+// NewAllowedMoneyMarket returns a new AllowedMoneyMarket
+func NewAllowedMoneyMarket(denom string, bl, sm, cf, irm, rf, kr bool) AllowedMoneyMarket {
+	return AllowedMoneyMarket{
+		Denom:                  denom,
+		BorrowLimit:            bl,
+		SpotMarketID:           sm,
+		ConversionFactor:       cf,
+		InterestRateModel:      irm,
+		ReserveFactor:          rf,
+		KeeperRewardPercentage: kr,
+	}
+}
+
+// Allows implement permission interface
+func (amm AllowedMoneyMarket) Allows(current, incoming hard.MoneyMarket) bool {
+	allowed := ((amm.Denom == current.Denom) && (amm.Denom == incoming.Denom)) &&
+		((current.BorrowLimit.Equal(incoming.BorrowLimit)) || amm.BorrowLimit) &&
+		((current.SpotMarketID == incoming.SpotMarketID) || amm.SpotMarketID) &&
+		((current.ConversionFactor.Equal(incoming.ConversionFactor)) || amm.ConversionFactor) &&
+		((current.InterestRateModel.Equal(incoming.InterestRateModel)) || amm.InterestRateModel) &&
+		((current.KeeperRewardPercentage.Equal(incoming.KeeperRewardPercentage)) || amm.KeeperRewardPercentage)
+	return allowed
```

Comment:
I'd add `ReserveFactor` equality check in case we ever need to change it via committee.